language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/FailStatementNodeContext.java

```diff
@@ -57,4 +67,67 @@ public FailStatementNodeContext() {
         
         return completionItems;
     }
+
+    @Override
+    public void sort(BallerinaCompletionContext context, FailStatementNode node,
+                     List<LSCompletionItem> completionItems) {
+        for (LSCompletionItem lsCItem : completionItems) {
+            if (lsCItem.getType() == LSCompletionItem.CompletionItemType.SYMBOL) {
+                SymbolCompletionItem symbolCompletionItem = (SymbolCompletionItem) lsCItem;
+                if (isCompletionItemSubTypeOfError(symbolCompletionItem)) {
```

Comment:
If we merge this condition with line 75, the else clause would drop to the outermost else, right? What if we do that since we set priority 3 in both cases?

Reply:
+1